### PR TITLE
add pydflow

### PR DESCRIPTION
**Pull request by njzjz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pydflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  license_family: LGPL
+  license_file: LICENSE
+  summary: Dflow is a Python framework for constructing scientific computing workflows employing Argo Workflows as the workflow engine.
+  doc_url: https://github.com/deepmodeling/dflow
+  dev_url: https://github.com/deepmodeling/dflow
+
+extra:
+  recipe-maintainers:
+    - njzjz
+    - zjgemi
```
Comment (njzjz) [inline recipes/pydflow/meta.yaml:55]: @zjgemi Please confirm here you are willing to be listed here.

Comment (zjgemi) [reply recipes/pydflow/meta.yaml:55]: I agree to be a maintainer.

**Comment by njzjz** [comment]
@conda-forge/help-python ready for review

File: recipes/pydflow/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - argo-workflows 5.0.0
+    - jsonpickle
+    - minio
+    - kubernetes
+    - pyyaml
+    - cloudpickle 2.2.0
+    - requests
+    - tqdm
+
+test:
```
Comment (synapticarbors) [inline recipes/pydflow/meta.yaml:40]: I recommend adding a `pip check` test:

https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L61-L64

**Review by synapticarbors — COMMENTED** [review]
This looks good. I just had the single comment. Since it's been a while you might also want to pull in changes from the upstream `main` into your branch. Thanks for your patience. I realize that you requested a review but didn't get one.

File: recipes/pydflow/meta.yaml (hunk 2 above)
Comment (njzjz) [reply recipes/pydflow/meta.yaml:40]: Done!

**Pull request merged**